src/Tizen.Applications.EventManager/Tizen.Applications/ApplicationEventManager.cs

```diff
@@ -0,0 +1,109 @@
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// public class EventManagerSample
+    /// {
+    ///     /// ...
+    ///     Bundle bundle = new Bundle();
+    ///     bundle.AddItem("key", "value");
+    ///     EventManager.PublishAppEvent("event.org.example.helloworld.event, bundle);
+    /// }
+    /// </code>
+    /// </example>
+    /// <since_tizen> 6 </since_tizen>
+    public static class ApplicationEventManager
+    {
+        /// <summary>
+        /// Sends the User-Event to receiver applications.
+        /// </summary>
+        /// <param name="eventName">The event's name to send.</param>
+        /// <param name="eventData"> The event's data to send.</param>
+        /// <remarks>The format of User-Event's name MUST be "event.{sender's appid}.{user-defined name}", refer to 'The name-format of User-Event' section,
+        /// If the event_name is invalid, throw InvalidOperationException</remarks>
+        /// <exception cref="ArgumentException">Thrown in case of an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Publish(string eventName, Bundle eventData)
+        {
+            Interop.AppEvent.ErrorCode err = Interop.AppEvent.EventPublishAppEvent(eventName, eventData.SafeBundleHandle);
+            if (err != Interop.AppEvent.ErrorCode.None)
+            {
+                ErrorFactory.ThrowException(err, "Publish event");
+            }
+        }
+
+        /// <summary>
+        /// Sends the User-Event to trusted receiver-applications.
+        /// </summary>
+        /// <param name="eventName">The event's name to send.</param>
+        /// <param name="eventData"> The event's data to send.</param>
+        /// <param name="isTrusted"> Whether the event is trusted or not.</param>
+        /// <remarks>The application which has same certification with sender can receive the event.
+        /// The format of User-Event's name MUST be "event.{sender's appid}.{user-defined name}", refer to 'The name-format of User-Event' section,
+        /// If the event_name is invalid, throw InvalidOperationException</remarks>
+        /// <exception cref="ArgumentException">Thrown in case of an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Publish(string eventName, Bundle eventData, bool isTrusted)
+        {
+            if (isTrusted)
+            {
+                Interop.AppEvent.ErrorCode err = Interop.AppEvent.EventPublishTrustedAppEvent(eventName, eventData.SafeBundleHandle);
+                if (err != Interop.AppEvent.ErrorCode.None)
+                {
+                    ErrorFactory.ThrowException(err, "Publish trusted event");
+                }
+            }
+            else
```

Comment:
You can just call Publish(eventName, eventData);
```suggestion
            else
            {
                Publish(eventName, eventData);
            }
```

Reply:
Done